install/install.php

```diff
@@ -314,10 +314,10 @@ function pluginFusioninventoryInstall($version, $migrationname='Migration') {
                          array('mode'=>2, 'allowmode'=>3, 'logs_lifetime'=>30));
       CronTask::Register('PluginFusioninventoryAgent', 'cleanoldagents', (3600 * 24),
                          array('mode' => 2, 'allowmode' => 3, 'logs_lifetime' => 30,
-                               'comment'=>'Clean agents not contacted since xxx days'));
+                               'comment'=> __('Clean agents not contacted since xxx days', 'fusioninventory')));
```

Comment:
I suppose that it's better to rename the string in "Delete agent that have not contacted the server since xxx days".